src/XML.php

```diff
@@ -125,7 +125,7 @@ public function DoXML()
                 $this->ErrorString = "Error the query can't be a null string";
                 return -1;
             }
-            $result = $DB->query($strqry);
+            $result = $DB->doQuery($strqry);
```

Comment:
Maybe this code miss a test? (out of scope of current PR anyway)

Reply:
See #14323. It is probably not used by any plusing since a while. Removal has been done in main just to ensure that it will not break any plugin that would still have a reference to it.